core/tests/test_utils.py

```diff
@@ -466,6 +466,26 @@ def get_updated_param_dict(
                 obj_type, new_param_dict)
         return new_param_dict
 
+    def get_cache_slug_filepath(self):
```

Comment:
Is this needed at all? It seems to repeat logic that's already in utils.py...

Instead, could get_static_asset_url be used in all circumstances, in order to avoid duplication?